routes/Nova/api.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use GreenBar\MenuBuilder\Nova\Http\Controllers\MenuBuilderController;
+
+/*
+|--------------------------------------------------------------------------
+| Tool API Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you may register API routes for your tool. These routes
+| are loaded by the ServiceProvider of your tool. They are protected
+| by your tool's "Authorize" middleware by default. Now, go build!
+|
+*/
+
+Route::get('/dashboard', MenuBuilderController::class . '@index');
+
+// Menu Item Operations
+Route::get('/list-menu-items/{menu_id}', MenuBuilderController::class . '@list_menu_items');
+Route::get('/save-menu-order/{menu_id}', MenuBuilderController::class . '@save_menu_order');
```

Comment:
If this route is meant to save something then should it be a "post" instead of a "get?"

Reply:
yes it should...